sonic-psud/scripts/psud

```diff
@@ -606,25 +596,14 @@ class DaemonPsud(daemon_base.DaemonBase):
             self.psu_chassis_info = PsuChassisInfo(SYSLOG_IDENTIFIER, platform_chassis)
 
         self.psu_chassis_info.run_power_budget(chassis_tbl)
-
-    def update_master_led_color(self, chassis_tbl):
-        if not platform_chassis or not self.psu_chassis_info:
-            return
-
-        psu_chassis_info = self.psu_chassis_info
-        if psu_chassis_info.update_master_status():
-            log_on_status_changed(self, psu_chassis_info.master_status_good,
-                                  'PSU supplied power warning cleared: supplied power is back to normal.',
-                                  'PSU supplied power warning: {}W supplied-power less than {}W consumed-power'.format(
-                                      psu_chassis_info.total_supplied_power, psu_chassis_info.total_consumed_power)
-                                  )
-            psu_chassis_info._set_psu_master_led(psu_chassis_info.master_status_good)
+        self.psu_chassis_info.update_psu_chassis_info()
```

Comment:
This should be self.psu_chassis_info.update_master_status()

Reply:
Thanks for the catch! Fixed in latest commit. Please test and review again.

Reply:
@mprabhu-nokia: Can you test and review again, please?

Reply:
Yes, looks good.

Reply:
Thank you!